fastlane/Fastfile

```diff
@@ -326,21 +326,26 @@ lane :validate_docs do
 
   # Test if generating the docs is successful
   clone_docs do
+    # Test if generating the docs is successful
     generate_markdown_docs(target_path: ".")
+    # Verify docs tests are still passing
+    run_docs_tests
   end
-
-  # Verify docs are still working
-  verify_docs
 end
 
 desc "Makes sure the tests on https://docs.fastlane.tools still work with the latest version"
 lane :verify_docs do
   clone_docs do
-    Bundler.with_clean_env do
-      puts(`sed -i -e "s/activate_bin_path/bin_path/g" $(which bundle)`) # workaround for bundler https://github.com/bundler/bundler/issues/4602#issuecomment-233619696
-      sh("bundle install")
-      sh("bundle exec fastlane test skip_building:true") # skip_building since we don't have a proper python environment set up
-    end
+    run_docs_tests
+  end
+end
+
+desc "Makes sure the tests on https://docs.fastlane.tools still work with the latest version"
+private_lane :run_docs_tests do
```

Comment:
Even though this does look good in its own lane I don't think it 100% works since this is required to run within the context of the `clone_docs` block (because it needs to run in a clone docs directory). 

If we move the `generate_markdown_docs(target_path: ".")` call within the `verify_docs` lane (like mentioned above), we can move this code back into `verify_docs` and not have any duplicate code

Reply:
That's why its a private lane that can not be run by itself. Other comments see above.

Reply:
I think I'm fine with leaving this as is then but maybe add a comment that saying this needs to run in the directory/scope of the docs directory? My biggest concern is that this can **ALSO** run "successfully" in the scope of this directory because there is also a `test` lane. That is why I would prefer if we could keep this code inside of a `clone_docs` block 🙃

Reply:
Can you just make the necessary changes in this PR? I'm totally fine with cleaning up a little here, but I am not 100% I would solve it correctly.

Reply:
Waiting for you on this one @joshdholtz